package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/skyux-builder-plugin-auth-email-whitelist",
```

Comment:
Do we want to republish this under the `@skyux-sdk` scope?  Related question for @Blackbaud-SteveBrush, did we update the plugin logic to accept both of those, or is the intention to load plugins from the `@blackbaud`scope?

Reply:
We can, but it does work as-is.  I tested it on a SKY UX 3 SPA and it got pulled in as expected.